### DISCUSSION TITLE
Open link in default browser instead of webview2

### DISCUSSION THREAD

**Discussion by Komas19-new** [MicrosoftEdge/WebView2Feedback#4003] [Q&A]
So, i have an win form app, that uses webview2, and i have the url of my website, in source in the properties of webview21, and inside my website, there is an buy button, that opens an new tab, but it opens in webview2, instead of inside the default browser. I want it to be like that since it redirect you to another website, not owned by me, that requires you to login, that would be annoying to relogin, while the default browser is already logged in.

**Comment by victorthoang** [comment]
Hello @Komas19-new,

Thanks for your inquiry! I've assigned this to a dev that might best answer your question.

**Comment by LiangTheDev** [comment]
You can handle [NavigationStarting](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.navigationstarting?source=recommendations&view=webview2-dotnet-1.0.1462.37) event, check the url to decide whether you want to launch in browser. When decided to launch in browser, then cancel the navigation and call OS API to launch the url in default browser.
